go/libraries/doltcore/env/actions/commitwalk/commitwalk.go

```diff
@@ -151,21 +149,29 @@ func (q *q) Get(ctx context.Context, ddb *doltdb.DoltDB, id hash.Hash) (*c, erro
 
 	commit, ok := optCmt.ToCommit()
 	if !ok {
+		// TODO: add this to loaded?
```

Comment:
What does this mean? I see the `q.loaded` map, but I'm looking for more context. Should we do something as part of this PR? If we're going to keep a TODO in here, it should be explained more so people can understand what it is suggesting and why. Otherwise we should remove it.